doc/CONTRIBUTING.md

```diff
@@ -106,6 +108,13 @@ Don't worry about setting DynamoDB endpoints or creating tables: the broker's co
 
 **Note:** The local version of DynamoDB is not recommended for production.
 
+### Install RabbitMQ
+
+RabbitMQ is used to pass jobs to the validator, and requires Erlang to be installed before RabbitMQ.
+
+1.  Install Erlang based on the [download instructions](https://www.erlang.org/downloads)
+2.  Choose an installation guide based on your OS for [RabbitMQ](https://www.rabbitmq.com/download.html).  Be sure to install Erlang before installing RabbitMQ.  The default user and password is "guest"/"guest", be sure to update the config.yml file if you change this.
```

Comment:
Should we set the RabbitMQ username/pwd in the sample config file to guest/guest, since we're telling folks they'll only have to make a change if they update these values when installing?